cli-commands/pg/post/show.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+UbiRodish.on("pg").run_on("show") do
+  fields = %w[id name state location vm_size storage_size_gib version ha_type flavor connection_string primary earliest_restore_time latest_restore_time firewall_rules metric_destinations ca_certificates].freeze.each(&:freeze)
```

Comment:
Can we read `fields` from openapi.yml somehow?

Otherwise, when we introduce new fields, we need to remember to add them in multiple places.

Reply:
I don't think the additional complexity inherent in that approach is worth the benefit.